Page/Cart/cart.html

```diff
@@ -0,0 +1,161 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/style.css">
+    <link rel="stylesheet" href="cart.css">
+    <script src="js/fontawesomepro.js"></script>
+    <link rel="stylesheet" href="https://site-assets.fontawesome.com/releases/v6.0.0/css/all.css">
+    <title>Scentcy</title>
+    <link rel="stylesheet" href="https://robust-ui.netlify.app/components/Card/card.css" />
+    <link rel="stylesheet" href="https://robust-ui.netlify.app/components/Button/button.css" />
+</head>
+
+<body>
+    <header>
+        <nav class="nav-container-box">
+            <div class="nav-logo"><a href="/../index.html">Scentcy <i class="fa-solid fa-spray-can-sparkles"></i></a></div>
+            <input type="search" placeholder="Search for items here" class="nav-input">
+            <div class="navbar-pills"> 
+              <a class="buy-icon" href="/Page/Wishlist/wishlist.html"><i class="fa-solid fa-heart"></i></a>
+              <a class="buy-icon" href="/Page/Cart/cart.html"><i class="fa-solid fa-cart-plus"></i></a>
+              <a class="buy-icon" href="/page/logout/logout.html"><i class="fa-solid fa-right-from-bracket"></i></a>
+            </div>
+        </nav>
+    </header>
+
+        <main>
+            
+            <div class="cart-heading">My Cart</div>
+
+<div class="divider-space"></div>
+
+            <div class="cart-container-box">
+                <div class="cart-cards">
+
+
+
+                    <div class=" card card-price">
+
+                        <div class="image-product">
+                            <img src="/images/perfume-one.PNG"
+                                alt="Perfume" />
+                                <div class="card-dismiss">
+                                    <i class="fa-regular fa-circle-xmark"></i>
+                                </div>
+                        </div>
+                        <div class="card-title">
+                           DavidOff
+                        </div>
+                        <div class="card-text">
+                            Cool Water
+                        </div>
+                        <div class="all-price">
+                            <span class="price-after">
+                                Rs.4,212
+                            </span>
+                            <span class="price-before">
+                                Rs.5,200
+                            </span>
+                            <span class="discount">
+                                (19% OFF)
+                            </span>
+                        </div>
+
+                         <button class="btn btn-primary"><i class="fa-solid fa-cart-plus"></i> ADD TO
+                            CART</button>
+                        <button class="btn btn-error"><i class="fa-solid fa-xmark"></i>
+                            Remove</button>
+                      
+                    </div>
+
```

Comment:
can remove the extra spaces in between the code

Reply:
done